MapboxCoreNavigation/OfflineDirections.swift

```diff
@@ -138,3 +176,15 @@ fileprivate func currentQueueName() -> String? {
     let name = __dispatch_queue_get_label(nil)
     return String(cString: name, encoding: .utf8)
 }
+
+extension URL {
+    
+    func fileSize() -> UInt64? {
```

Comment:
computed property, not function.

Reply:
We talked about functions indicating a non-trivial calculation vs computed property and since this is I/O, I went with function. However, I changed this to a computed property.